### ISSUE TITLE
Configuration only sensor proposal.

### ISSUE THREAD

**Issue by amzaldua** [martijnvwezel/watermeter-esphome#12]
# Problem/Motivation

> (Why the issue was filed)

## Expected behavior

> (What you expected to happen)

## Actual behavior

> (What actually happened)

## Steps to reproduce

> (How can someone else make/see it happen)

## Proposed changes

Are you considering launching a module sensor for external connection to a standard ESP32 board? I favor this solution because the location where the device is installed is accessible to individuals who are not residents of my home

**Comment by martijnvwezel** [comment]
So you want a module sensor for external connection to a standard esp32 board? There is on this board already a standard esp32? Dont get what you want.

**Comment by amzaldua** [comment]
Yes, that's correct. I would like a sensor module (just the sensor) to connect to an external ESP32 board. This way, your excellent solution would be compatible with more brands of water meters, and for meters located outside of houses, it would be more discreet and safe.

**Comment by martijnvwezel** [comment]
l will do this possibly in the future. Currently, focussing on Matter implementation and after that I can have a look in a special sensor board. Only thing to figure out is how to montage it..

**Comment by martijnvwezel** [comment]
What brand of wateermeter do you have?

**Comment by amzaldua** [comment]
JANZ JV600
![IMG_3546](https://github.com/martijnvwezel/watermeter-esphome/assets/116205677/2923ba25-7068-4295-afae-5d383f5d269d)

**Comment by martijnvwezel** [comment]
![image](https://github.com/martijnvwezel/watermeter-esphome/assets/4363788/0d6ed7fb-9afe-42d4-977a-ef94397b842d)

![image](https://github.com/martijnvwezel/watermeter-esphome/assets/4363788/601ac3f0-03be-492e-a4d7-774de095176e)

one of the design concepts, actually working on a sensor version and small sensor version with USB-C

**Comment by amzaldua** [comment]
Awesome! Is the microcontroller included, right? This sensor only needs to be plugged into a USB power supply, I suppose...

**Comment by marco12en3** [comment]
It should have 2 ways of connecting, usb or a pinheader. With usb cdc every time the amount of liters changes you send an update over serial. With the pinheader you should be able to power the board and get a rotary encoder output and/or a uart output.

**Comment by martijnvwezel** [comment]
A small update, I rejected the board design given above. The following board is going to be produced:
![front](https://github.com/martijnvwezel/watermeter-esphome/assets/4363788/0d7b63c1-16d8-4257-b17d-9aa11a56f3e8)
![back](https://github.com/martijnvwezel/watermeter-esphome/assets/4363788/a0de2a8b-80cb-4860-960c-518d50e67d98)

I think in 2 weeks they will be available to be shipped!

**Comment by martijnvwezel** [comment]
@amzaldua  using the sticky double sided tape you can tape it to your watermeter!

**Comment by martijnvwezel** [comment]
https://www.tindie.com/products/muino/3-phase-muino-light-sensor-encoder/?utm_source=mandrill&utm_medium=email&utm_campaign=product_approved

**Comment by martijnvwezel** [comment]
@amzaldua  you can order it now from the store!

**Comment by martijnvwezel** [comment]
![circle_tape](https://github.com/martijnvwezel/watermeter-esphome/assets/4363788/894c2b34-55aa-4bf4-b387-6c558c7b6e7c)
![front_real](https://github.com/martijnvwezel/watermeter-esphome/assets/4363788/ae6f5220-5ddc-475a-9ffa-e1e31bb7a853)

**Comment by martijnvwezel** [comment]
the sensor arrived!
